src/main/java/com/adobe/cq/testing/client/CQAssetsClient.java

```diff
@@ -236,6 +244,20 @@ public String getAssetProcessingFailures(String assetPath) throws ClientExceptio
 
     public CQAssetsClient(CloseableHttpClient http, SlingClientConfig config) throws ClientException {
         super(http, config);
+
+        // same settings as SlingClient
+        storageClient = HttpClientBuilder.create()
+            .useSystemProperties()
+            .setUserAgent("Java")
+            // Connection
+            .setMaxConnPerRoute(10)
+            .setMaxConnTotal(100)
+            // Interceptors
+            .addInterceptorLast(new TestDescriptionInterceptor())
```

Comment:
@alexkli not sure if this interceptor is useful in this case. Would the headers `X-Sling-TestClass` and `X-Sling-TestName` be used somewhere in requests to azure? Anyway, it probably doesn't hurt.

Reply:
Yeah, might not be fully needed, but since they are custom headers (`X-*`) they should not hurt. And maybe there is some request logging somewhere some time where it might be nice to trace things back to the unit test this way.